.github/scripts/submit-provider-key.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ -z "${BODY}" ]]; then
+  echo "Please run this script from a GitHub Action."
+  exit 1
+fi
+if [[ -z "${TITLE}" ]]; then
+  echo "Please run this script from a GitHub Action."
+  exit 1
+fi
+if [[ -z "${NUMBER}" ]]; then
+  echo "Please run this script from a GitHub Action."
+  exit 1
+fi
+if [[ -z "${GH_USER}" ]]; then
+  echo "Please set GH_USER"
+  exit 1
+fi
+
+namespace=$(echo "${BODY}" | grep "### Provider Namespace" -A2 | tail -n1 | tr "[:upper:]" "[:lower:]" | sed -e 's/[\r\n]//g')
+providername=$(echo "${BODY}" | grep "### Provider Name" -A2 | tail -n1 | tr "[:upper:]" "[:lower:]" | sed -e 's/[\r\n]//g')
+keydata=$(echo "${BODY}" | grep -A 1000 "BEGIN PGP PUBLIC KEY BLOCK"  | grep -B 1000 "END PGP PUBLIC KEY BLOCK")
+echo "${keydata}" > tmp.key
+
+if [[ ! "${namespace}" =~ ^[a-zA-Z0-9-]+$ ]]; then
+  gh issue comment "${NUMBER}" -b "Failed validation: Invalid namespace: '${namespace}'"
+  exit 1
+fi
+
+if [[ "${providername}" = "*No response*" ]]; then
+  providername=""
+fi
+if [[ -n "${providername}" ]]; then
+  if [[ ! "${providername}" =~ ^[a-zA-Z0-9-]+$ ]]; then
+    gh issue comment "${NUMBER}" -b "Failed validation: Invalid provider name: '${providername}}'"
+    exit 1
+  fi
+fi
+
+set +e
+go run ./cmd/verify-gpg-key -org "${namespace}" -username "${GH_USER}" -key-file=tmp.key -output=./output.json
+verification=$?
+set -euo pipefail
+
+gh issue comment "${NUMBER}" -b "$(jq -r '.' < ./output.json || true)"
+if [[ "${verification}" != 0 ]]; then
+  exit 1
+fi
+
+if [[ -z "${providername}" ]]; then
+  keyfile="../keys/${namespace:0:1}/${namespace}/provider-$(date +%s).asc"
+else
+  keyfile="../keys/${namespace:0:1}/${namespace}/${providername}/provider-$(date +%s).asc"
+fi
+if [[ -d "$(dirname "${keyfile}")" ]]; then
+  msg=Updated
```

Comment:
```suggestion
  msg="Updated"
```
If i'm being really syntax strict, not really a problem but nice to do